corehq/motech/repeaters/migrations/0016_add_indexes.py

```diff
@@ -21,7 +21,18 @@ class Migration(migrations.Migration):
                     index=models.Index(
                         condition=models.Q(("is_deleted", False)),
                         fields=["id"],
-                        name="is_deleted_partial_idx",
+                        name="deleted_partial_idx",
+                    ),
+                ),
+                migrations.AddIndex(
+                    model_name="repeater",
+                    index=models.Index(
+                        condition=models.Q(
+                            ("is_deleted", False),
+                            ("is_paused", False),
+                        ),
+                        fields=["id"],
+                        name="deleted_paused_partial_idx",
```

Comment:
No issues with this, but just curious if it would make sense to name this index something like "active_partial_idx" instead, since it effectively only contains active repeaters? Again, no strong preference, just curious what the goal of an index name should be. Though with this logic it might be hard to give the two different indices meaningful names.

Reply:
I'm not sure what the goal of an index name is, but I know that we're happy to go with whatever Django decides nearly all the time. :) I had to change it from "is_deleted_is_paused_partial_idx" (iirc) because the name was too long.

:+1: I agree, "active_partial_idx" is a better name.

Reply:
8888fff8570